Add Remote page (#1408)
EuroPython/website: src/content/pages/remote.mdx

---
title: EuroPython 2025 Remote
subtitle: Join EuroPython 2025 remotely!
toc: true
---

import { Image } from "astro:assets";
import discord from "./images/discord.png";
import remote from "./images/remote.png";

# EuroPython 2025 Remote

As happy as we are about finally being able to see many friends in [Prague](/prague)
this year, we are also aware that many of you will not be able to make it in-person.
Again, we present you EuroPython 2025 remote:
Wherever you are based, let's celebrate Python and our community, rejoice at our diversity,
learn and have fun in Prague and beyond! In this spirit, we strive to make the remote participation
as interactive as possible.
<Image src={remote} width="1200" alt="" />

## How to participate in the conference remotely?

- To participate in the conference interactively (and have guaranteed access to livestreams), a "remote ticket" registration is required. See details and pricing on the [Tickets information](/tickets#remote-tickets) page.

> - All ticket holders will receive a link to join our [Discord server](/discord) for the conference.

Reviewer: Let's remove this as well, just so we have less random people joining the server without the tickets :sweat_smile:

<Image src={discord} width="1200" alt="" />

<Note>in-person ticket holders will be provided with remote event access as well.</Note>

## What is included in the remote ticket?
Remote participation between 16-18 July (Wednesday - Friday):
- Livestreams of Keynotes, panel discussions, all 6 tracks of talks.
- Live participation (text-based) on Q&A session post-talks.
- Text-based discussions with available speakers.
- Online job board and chat with sponsors at the remote booths.

> - On-Demand viewing, which will be available hours after the live stream.

Reviewer 2: ```suggestion
- On-demand viewing, which will be available hours after the live stream.
```

> <Note type="warning">Tutorials and Workshops on 8 and 9 July will not be streamed.</Note>

Reviewer 2: ```suggestion
<Note type="warning">Tutorials and Workshops on 14 and 15 July will not be streamed.</Note>
```

Author: I commit that change already.

## What changes ###
This year, we will **NOT** provide open and free access to the livestreams of our conference talks (16-18 July).
If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
Purchasing a remote ticket not only gives you access to the conference but also helps support the community!

## Livestreams without interaction ###
As per our tradition, we also provide open and free access to the livestreams of our conference talks (19-21 July). You are not required to register a ticket, but you are welcome to if you would like an email reminder or receive up-to-date information about the conference.

> But The free livestream-only ticket does not include support and you will be unable to interact with the speakers or other in-person audience. Consider getting a remote ticket instead.

Reviewer 2: ```suggestion
```
this unfortunately does not apply this year, so should be deleted :(

{/*
## YouTube Links Every Day ###

> <Button text="Enjoy all EuroPython 2025 Livestreams" url="https://youtube.com/playlist?list=PL8uoeex94UhFzv6hQ_V02xfMCcl8sUr4p" >Why list many links when you have a playlist?</Button>

Reviewer 2: Since livestream is not open, there won't be a need to add the button during the conference.

Author: OK, I will remove it.
